### ISSUE TITLE
I can't see my maze in the list

### ISSUE THREAD

**Issue by Hypo74** [beam41/openrct2-maze-ganerator#1]
Hi,

I tried your plugin, I partially built a maze, I drawed the path around it. When I try to open the menu of the mod, I just cannot see my maze in the list.
![mazepb](https://user-images.githubusercontent.com/122159876/231829319-c7c91b6f-9436-4f9e-bd3c-f7a85e272102.png)
Thanks!

**Comment by beam41** [comment]
Hey, thank for trying my plugin.

I cannot replicate this bug tho. Can I have the save file?

**Comment by Hypo74** [comment]
Hi,

Fun fact, I tried a few things and I found the glitch. I switched my game from french to english and now I see my maze in the list.
I tried to switch back again to french, it diseappeared (whether my maze is called "maze 1" or "labyrinthe 1".
![mazepb](https://user-images.githubusercontent.com/122159876/232009355-99758c87-bc62-40da-9bad-068a398125b7.png)

So yeah, maybe it comes from the way your plugin searchers for mazes.
